### DISCUSSION TITLE
Shader incompatibility

### DISCUSSION THREAD

**Discussion by doemty** [UltimatChamp/optimum-fuse#20] [Q&A] [answered]
What mod does add these effects?
![2024-05-26_04 52 09](https://github.com/UltimatChamp/optimum-fuse/assets/151661389/a7c914c6-2670-4657-92e2-19bf538a5eca)

**Comment by UltimatChamp** [comment]
Does removing **Fancy Block Particles Renewed** mod helps?

**Reply by doemty** [reply]
Yeah, thank you!

**Reply by UltimatChamp** [reply, marked as answer]
Ok! Then, I'll release a version today to fix the issue... And configure the mod be less annoying!

**Comment by UltimatChamp** [comment]
Fixed in 1.10.0-rc10!
